.eslintrc

```diff
@@ -0,0 +1,44 @@
+{
+    "extends": "airbnb",
+    "parser": "babel-eslint",
+    "overrides": [
+        {
+            "files": [
+                "src/**/*.js",
+                "src/**/*.jsx"
+            ],
+            "rules": {
+                "indent": ["error", 4],
+                "react/jsx-indent": ["error", 4],
+                "react/jsx-indent-props": ["error", 4],
+                "react/jsx-filename-extension": ["error", {"extensions": [".jsx", ".js"]}],
+                "max-len": ["error", 120],
+                "no-plusplus": ["off"],
+                "react/jsx-curly-newline": ["error", {
+                    "singleline": "forbid",
+                    "multiline": "require"
+                }],
+                "operator-linebreak": ["error", "after", {
+                    "overrides": {
+                        "?": "before",
+                        ":": "before"
+                    }
+                }],
```

Comment:
Personally I'm in favor of `before`. IMO operators are added because the second expression is necessary, so they should go on the same line as the second expression rather than the first. Plus that puts all the operators in the same column which is nice. Thoughts on this?

([Link to the docs](https://eslint.org/docs/rules/operator-linebreak) in case anybody else wants to see what they think about it.)

Reply:
Personally I've never found the operators being in the same column to really offer any sort of advantage, except maybe readability in a few edge cases (that I've never personally seen in the wild). For me, the existence of the operator on the previous line causes me to realize that the line carries over without having to look at the next line, as there are plenty of cases where the expression is valid without the continuation

Reply:
Ultimately I don't think this makes a huge difference, so I won't choose this hill to die on. You've got more experience with this language than I so I'll defer to you here.